nipype/interfaces/ants/registration.py

```diff
@@ -1034,3 +1034,115 @@ def _list_outputs(self):
         if len(self.inputs.save_state):
             outputs['save_state'] = os.path.abspath(self.inputs.save_state)
         return outputs
+
+
+class MeasureImageSimilarityInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(
+         2, 3, 4,
+         argstr='--dimensionality %d', usedefault=False,
```

Comment:
I believe the preference is not to explicitly set parameters with their defaults, so I'd remove `usedefault=False`.

`usedefault=False`